docs/rhoas/rhoas-helm-guide/README.adoc

```diff
@@ -0,0 +1,395 @@
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-helm-guide-rhoas-cli"]
+= Configuring OpenShift Application Services in Helm
+ifdef::context[:parent-context: {context}]
+:context: helm-guide-rhoas-cli
+
+// Purpose statement for the assembly
+[role="_abstract"]
+In this guide we are going to explain how to configure Application services in your Helm deployment. For more information about helm, please refer to the official docs.
+
+A developer can have his application deployed in a Kubernetes platform such as Red Hat OpenShift in various ways - application deployed as a Kubernetes deployment or application packaged as a helm chart. We will walk through how to connect the applications to Managed Services for both the scenarios.
+
+To connect an application deployed in OpenShift to the Managed Services, we need to supply connection configurations and credentials. RHOAS CLI simplifies this task for us by generating connection configurations and credentials as Kubernetes objects.
+
+.Prerequisites
+ifndef::community[]
+* You have a Red Hat account.
+endif::[]
+* You have a running Kafka instance in {product-kafka}.
+* You've installed the latest version of the `rhoas` CLI. See {base-url}{installation-guide-url-cli}[Installing and configuring the rhoas CLI^].
+* You've installed https://helm.sh/docs/intro/quickstart/:[Helm CLI^] version 3.9.0 or above
+* You've installed https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html:[OpenShift CLI] version 4.8.5 or above
+
+[id="proc-generating-configmap-for-application-services_{context}"]
+== Generate ConfigMap for Application Service Instances
+
+The application deployed in the Kubernetes platform needs to connect to the Kafka instance. To achieve this, one needs to supply the URL to the Kafka Instance. The URLs being non-confidential data can be supplied as Kubernetes ConfigMap. The helm chart can refer to the ConfigMap deployed in the OpenShift cluster which will henceforth connect the application to the Kafka Instance.
+
+This is implemented by RHOAS CLI using service contexts. In OpenShift Application Services, a service context is a defined set of instances running in cloud services such as OpenShift Streams for Apache Kafka and OpenShift Service Registry. You might create different contexts for specific use cases, projects, or environments. To learn more about contexts, you can go through this https://github.com/redhat-developer/app-services-guides/tree/main/docs/rhoas/rhoas-service-contexts:[guide].
```

Comment:
Some suggested updates:

 - Application Services
 - To learn more about contexts, see...